content/sensu-go/6.2/observability-pipeline/observe-schedule/metrics.md

```diff
@@ -0,0 +1,365 @@
+weight: 33
+version: "6.2"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.2:
+    parent: observe-schedule
+---
+
+Sensu Go offers first-class built-in support for collecting and processing metrics for your entire infrastructure.
+
+In Sensu, metrics are events that contain metrics data.
+Sensu metrics can be metrics-only events created with data from the [Sensu StatsD listener][2] or [Prometheus][7].
+They can also be status and metrics events from [check output metric extraction][4].
+
+## Extract metrics from check output
+
+The Sensu agent can extract metrics data from check command output and populate an event's [metrics attribute][5] before sending the event to the Sensu backend for [processing][11].
+
+To extract metrics from check output, the check must include an [`output_metric_format` attribute][10] that specifies one of Sensu's [supported output metric formats][9].
+When a check includes the `output_metric_format` attribute, Sensu will extract the specified metrics from the check output and add them to the event data in the [metrics attribute][5].
+
+In addition, the check command must output metrics in the supported metrics formats.
+
+### Supported output metric formats
+
+Sensu supports the following formats for check output metric extraction.
+
+| Nagios             |      |
+---------------------|------
+output_metric_format | `nagios_perfdata`
+documentation        | [Nagios Performance Data][13]
+example              | {{< code plain >}}PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80{{< /code >}}
+
+| Graphite           |      |
+---------------------|------
+output_metric_format | `graphite_plaintext`
+documentation        | [Graphite Plaintext Protocol][14]
+example              | {{< code plain >}}local.random.diceroll 4 123456789{{< /code >}}
+
+| InfluxDB           |      |
+---------------------|------
+output_metric_format | `influxdb_line`
+documentation        | [InfluxDB Line Protocol][15]
+example              | {{< code plain >}}weather,location=us-midwest temperature=82 1465839830100400200{{< /code >}}
+
+| OpenTSDB           |      |
+---------------------|------
+output_metric_format | `opentsdb_line`
+documentation        | [OpenTSDB Data Specification][16]
+example              | {{< code plain >}}sys.cpu.user 1356998400 42.5 host=webserver01 cpu=0{{< /code >}}
+
+| Prometheus         |      |
+---------------------|------
+output_metric_format | `prometheus_text`
+documentation        | [Prometheus Exposition Text][17]
+example              | {{< code plain >}}http_requests_total{method="post",code="200"} 1027 1395066363000{{< /code >}}
+
+## Enrich metrics with tags
+
+[Output metric tags][1] are custom tags you can apply to enrich the metric points produced by check output metric extraction.
+
```

Comment:
I think something is missing here: need to explain where tag values come from if you *don't* use token substitution.

Reply:
I found it in the sensu-go issue: https://github.com/sensu/sensu-go/issues/2160#issuecomment-430678869